MIS/MISTests/MISTests.swift

```diff
@@ -2,7 +2,7 @@
 //  MISTests.swift
 //  MISTests
 //
-//  Created by Developer Admin on 06/08/20.
+//  Created by Wakralab on 06/08/20.
 //  Copyright © 2020 Developer Admin. All rights reserved.
```

Comment:
Developer Admin needs to be changed. App owner name should be given here which has to be set as the organisation. Created by can be Wakralab.